salt/apps/odlvideo/install.sls

```diff
@@ -0,0 +1,9 @@
+ensure_yarn_is_installed_for_odlvideo:
+  npm.installed:
+    - name: yarn
+    - user: root
+
+install_node_dependencies:
+  cmd.run:
+    - name: yarn install
+    - cwd: /opt/{{ salt.pillar.get('django:app_name') }}
```

Comment:
I don't see where the odl-video repo has been cloned to the `/opt/<app_name>` folder so that yarn can install what's in the lock file

Reply:
That gets handled by the Django formula in install.sls